src/m4/ax_python_devel.m4

```diff
@@ -186,34 +186,23 @@ variable to configure. See ``configure --help'' for reference.
 	   fi
 	fi
 
-	if test $ax_python_devel_found = yes; then
-	   #
-	   # Check if you have distutils, else fail
-	   #
-	   AC_MSG_CHECKING([for the sysconfig Python package])
-	   ac_sysconfig_result=`$PYTHON -c "import sysconfig" 2>&1`
-	   if test $? -eq 0; then
+	#
+	# Check if you have sysconfig, else fail
```

Comment:
Note that upstream `ax_python_devel` has support for the Debian layout (see my patches from 2022):

https://git.savannah.gnu.org/gitweb/?p=autoconf-archive.git;a=history;f=m4/ax_python_devel.m4

Reply:
It seem to me that both upstream ax_python.m4 and ax_python_devel.m4 uses the distutils python module we are trying to avoid.  I am thus reluctant to just upgrade to the latest upstream edition.

Reply:
Looking closer, I see that the distutils part is only a fallback, and the new version can be used without modifications.